### PR TITLE
Add Klantendag blog and update news items

### PR DESCRIPTION
**Pull request by lennartdefacto**
- [x] Add blog about november 5th klantendag
- [x] add images (social, mood, news, featured)
- [x] add blog to three featured items on homepage
- [x] remove now unused AVL image
- [x] Add additional images to blog

### CODE REVIEW

File: source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Na de opening was het woord aan Gerwin Veenstra, Productowner bij Defacto. Hij nam onze klanten mee in de techniek achter CAPP LMS, zoals 'continuous deployment', wat inhoudt dat we kleine verbeteringen en aanpassingen op continue wijze doorvoeren, zonder dat de klant hier altijd wat van merkt. Bij grotere ontwikkelingen en nieuwe features wordt dit uiteraard in overleg via onze consultants met onze klanten geregeld. Vervolgens ging het over de ontwikkeling van onze software. Zo zijn we bezig met het verbeteren van de zoekfunctie, zodat deze met een enkele zoekopdracht zowel de relevante resultaten bij artikelen, e-learning modules als videos laat zien. Een andere ontwikkeling is de feature om comments te plaatsen, bijvoorbeeld bij events. Hierdoor zullen gebruikers, die zich op hetzelfde event hebben ingeschreven, eenvoudig met elkaar kunnen communiceren.
```
Comment (RonaldDijkstra) [inline source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown]: Probably it's 'continu'

Comment (RonaldDijkstra) [inline source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown]: I would do: 

"We zijn onder andere bezig met het verbeteren van de zoekfunctie..."

File: source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+![Gerwin Veenstra](/images/blog/klantendag-005.jpg)
+
+Bij de tweede presentatie was het Miriam Boer, Academie Manager bij het Antoni van Leeuwenhoek instituut, die met veel enthousiasme vertelde over de implementatie van CAPP LMS en de samenwerking met Defacto. De rol en inzet van onze consultants werd geprezen, net als de manier waarop wij openstaan voor input als het gaat om softwareontwikkeling. Miriam sloot haar presentatie af door te zeggen dat, ondanks dat AVL als eerste CAPP LMS ging gebruiken, ze volledig achter de keuze staat en CAPP LMS van harte aanbeveelt bij klanten die nadenken over migratie en met name herimplementatie naar de laatste versie.
```
Comment (RonaldDijkstra) [inline source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown]: AVL is not the first one to use CAPP LMS.... 😆 prefer you reword this a bit.

File: source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Deze aanbeveling vormde een goede brug naar de laatste plenaire presentatie van de dag, door senior consultant Joep Lenglet. Hij vertelde over de mogelijkheden die de laatste versie van CAPP LMS biedt voor onze klanten en wat de verschillen tussen datamigratie en herimplementatie zijn. Zo werd voor de aanwezigen als snel helder dat het voor hun organisaties CAPP LMS pas echt optimaal ingezet kan worden bij herimplementatie.
```
Comment (RonaldDijkstra) [inline source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown]: So, what is that difference? Elaborate?

File: source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+![Joep Lenglet in gesprek met klanten](/images/blog/klantendag-009.jpg)
+
+Na de pauze werden kreeg iedereen de mogelijkheid om deel te nemen aan twee workshop rondes, met keuze uit drie workshops. Onze consultants hadden deze voorbereid en de onderwerpen waren: autorisatiestructuur, het gebruik van ons Kwaliteitspaspoort en vernieuwde e-learning en cursusmogelijkheden met CAPP LMS. Aan de hand van praktijkvoorbeelden kreeg men inzicht in de nieuwe en veranderde features bij CAPP LMS.
```
Comment (RonaldDijkstra) [inline source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown]: Reword workshop rondes?

Comment (RonaldDijkstra) [reply source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown]: Veranderde = verbeterde?

File: source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+Tot slot was er uiteraard de gelegenheid om gezamenlijk nog na te praten onder het genot van een borrel. Met dank aan de uitstekende catering van het AVL kon iedereen genieten van lekkere hapjes en drankjes. Onze consultants, account managers en ook een aantal developers waren druk in gesprek over CAPP LMS, ideeën voor de toekomst en allerlei andere onderwerpen.
```
Comment (RonaldDijkstra) [inline source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown]: minus uiteraard

Comment (RonaldDijkstra) [inline source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown]: minus ook

File: source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+![Iedereen borrelt nog even gezellig na.](/images/blog/klantendag-012.jpg)
```
Comment (RonaldDijkstra) [inline source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown]: Remove . from alt text

**Comment by RonaldDijkstra** [comment]
Images are too big in size, please optimise:

screenshot 2018-11-15 at 16 07 34

File: source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown (hunk 1 above)
Comment (lennartdefacto) [reply source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown]: Continue is an adjective in this case refering to 'wijze', that adds the extra 'e' on the end.

Comment (RonaldDijkstra) [reply source/blog/nl/2018-11-14-defacto-klantendag-november-2018.html.markdown]: Ok!

**Pull request merged**